Src/Plugins/Tests.Security/EndToEndTests.cs

```diff
@@ -30,20 +30,25 @@ protected EndToEndTests(ITestOutputHelper outputHelper) : base(outputHelper)
 
         protected abstract string Framework { get; }
 
+        protected abstract string PluginName { get; }
+
         protected override string TestLogResourceNameRoot => $"Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security.TestData.{TypeUnderTest}";
 
-        protected override string ProductDirectory => Path.Combine(base.ProductDirectory, @"Plugins\Tests.Security");
+        protected override string TestBinaryTestDataDirectory => Path.Combine(ProductRootDirectory, "src", "Plugins", TestBinaryName, "TestData");
```

Comment:
>TestBinaryTestDataDirectory [](http://example.com/codeflow?start=34&length=27)

We need to override this because the paths have another level called Plugins.